int32 num
float64 kl_score
float64 kl_x
float64 kl_y
float64 kl_z
float64 saiteki_kl
float64 sinrai_kl
